### DISCUSSION TITLE
Please contribute into XCTest framework

### DISCUSSION THREAD

**Discussion by AgranatMarkit** [pointfreeco/swift-issue-reporting#63] [Ideas]
Hi guys!

You have wonderful output that looks like a good contribution to XCTest framework.

Why don't you create pull request into https://github.com/apple/swift-corelibs-xctest ?

I think apple could use your implementation of https://github.com/pointfreeco/swift-custom-dump dump and it could nicely fit into https://github.com/apple/swift-corelibs-xctest/blob/main/Sources/XCTest/Public/XCTAssert.swift#L173 to get more useful output.

**Comment by stephencelis** [comment]
Hi @AgranatMarkit, I'm going to convert this to a discussion since we use issues for bug tracking.

The short of it is I don't think Apple takes contributions like this into frameworks that it also maintains internally. If you hear differently, though (by opening an issue or discussion with Apple), let us know!

**Reply by AgranatMarkit** [reply]
https://github.com/apple/swift-corelibs-xctest/issues/440#issue-1685326622
